README.md

```diff
@@ -99,6 +99,10 @@ CSS:
 D:
 - dmd
 
+Elixir:
+- credo
+- dogma
```

Comment:
You should add " (not enabled by default)" here, if it's really intentional.
Otherwise add it to `neomake#makers#ft#elixir#EnabledMakers`.